build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -254,6 +261,11 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
                 cmakeArgList.Add("-G");
                 cmakeArgList.Add(visualStudio);
                 
+                if (linkPortable)
+                {
+                    cmakeExtraArgs ="-DCLI_CMAKE_PORTABLE_BUILD=1";
```

Comment:
Why use `cmakeExtraArgs` here?  Why not just `cmakeArgList.Add` like all the code above this?